static/app/types/hooks.tsx

```diff
@@ -95,6 +101,7 @@ type FirstPartyIntegrationAdditionalCTAProps = {
  * Component wrapping hooks
  */
 export type ComponentHooks = {
+  'component:disabled-custom-symbol-sources': () => React.ComponentType<DisabledCustomSymbolSources>;
```

Comment:
The existing hook name was `custom-symbol-sources`, can we rather use that one? We shouldn't need a new hook.

Reply:
the existing one was updated to be this one